scm/github/webhook.go

```diff
@@ -184,9 +184,20 @@ func (c *client) processPREvent(h *library.Hook, payload *github.PullRequestEven
 		return &types.Webhook{Hook: h}, nil
 	}
 
-	// skip if the pull request action is not opened or synchronize
+	// if the edits to the PR are not related to the title or base branch, ignore them
+	if strings.EqualFold(payload.GetAction(), "edited") {
+		title := payload.GetChanges().GetTitle()
+		base := payload.GetChanges().GetBase()
+
+		if title == nil && base == nil {
+			return &types.Webhook{Hook: h}, nil
```

Comment:
Why? What if someone had a workflow that validated the PR description followed some format, would we not support that?